runtime/doc/api.txt

```diff
@@ -847,7 +847,8 @@ nvim_buf_line_count({buffer})                          *nvim_buf_line_count()*
                     {buffer}  Buffer handle
 
                 Return:~
-                    Line count
+                    Line count, or `0` if the buffer has been unloaded (see
```

Comment:
This section is autogenerated, please change the docstrings in api/buf.c only

Reply:
What command can I run to regenerate the api docs locally?

Reply:
`python3 scripts/gen_api_vimdoc.py`